### ISSUE TITLE
Dotted zero

### ISSUE THREAD

**Issue by orlp** [rsms/inter#445]
I like it when a font can disambiguate between zero and the letter o, but I'm not a huge fan of the slashed zero (in any font). Would it be possible to add a dotted zero variant instead?

I made a quick mockup of roughly what it could look like by copy/pasting in a period glyph in GIMP:

![image](https://user-images.githubusercontent.com/202547/159185384-d759337f-18c5-4f4d-9f96-b7b221ab6182.png)

**Comment by LoganDark** [comment]
There are even two variants of this, there is the dotted zero and the slitted zero. Slitted zero is like dotted, but it's a small vertical line instead (like a hyphen rotated 90 degrees). You see them sometimes in programming fonts like Input Mono.

**Comment by GabrielMiceli** [comment]
Dotted zeros are underrated in my opinion, and it always seems difficult for me to come by fonts that have it, so I would also very much like to see this included with Inter, even if it's an admittedly niche stylistic preference.

**Comment by jobcmax** [comment]
yes please!!!

**Comment by KrasnayaPloshchad** [comment]
I suggest this can be implemented as stylistic set (`ssXX`) feature.

**Comment by Lululalu** [comment]
Agreed. The slashed zero draw too much attention to itself. A dot is better.
